### PR TITLE
Selenium Basics (2) HW

### PR DESCRIPTION
**Pull request by katherinaGo**

### CODE REVIEW

File: Selenium_Basics/FindElementsTests.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    {
+        _driver.Navigate().GoToUrl(MainPageUrl);
+    }
+
+    [Test]
+    public void ListOfAvailableCountriesTest()
+    {
+        _driver
+            .FindElement(By.XPath(
+                "(//span[@class='top-navigation__item-link-holder--a11y']//preceding::span[@class='top-navigation__item-text'])[6]"))
```
Comment (dnnyhlwdplc) [inline Selenium_Basics/FindElementsTests.cs:18]: this locator is a little better :) using indexes is not a good practice :)
//*[contains(@class, 'top-navigation__item-link') and @href='/careers']

File: Selenium_Basics/FindElementsTests.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            .Click();
+
+        var actualListOfCountries = _driver
+            .FindElements(By.XPath("//*[@class='tabs__title' and contains(@role,'presentation')]"))
+            .Select(item => item.Text);
+        var expectedListOfCountries = new List<string> { "AMERICAS", "EMEA", "APAC" };
+        Assert.That(expectedListOfCountries, Is.EqualTo(actualListOfCountries),
+            "List of countries doesn't include: AMERICAS, EMEA, APAC");
```
Comment (dnnyhlwdplc) [inline Selenium_Basics/FindElementsTests.cs:26]: just a suggestion, you can write something like $"List of countries doesn't include: string.Join(',', expectedListOfCountries)"

File: Selenium_Basics/BaseTest.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace Selenium_Basics;
+
+public class BaseTest
```
Comment (dnnyhlwdplc) [inline Selenium_Basics/BaseTest.cs:7]: base test class? Great!

File: Selenium_Basics/FindElementsTests.cs [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    {
+        _driver
+            .FindElement(
+                By.XPath("//*[@class='header-search__button header__icon' and contains(@aria-expanded, 'false')]"))
+            .Click();
+        _driver.FindElement(By.XPath("//*[@id='new_form_search']")).SendKeys(searchWord);
+        _driver.FindElement(By.XPath("//*[@class='header-search__submit']")).Click();
+
+        var expectedResultSearchPage = _waiter
+            .Until(_driver => _driver.FindElement(By.XPath("(//*[@class='search-results__title'])[1]"))).Text;
```
Comment (dnnyhlwdplc) [inline Selenium_Basics/FindElementsTests.cs:75]: I am not sure if you need ()[1] in findElement method :)
if you need to get first element you can use axis [first()]

**Review by dnnyhlwdplc — APPROVED** [review]

File: Selenium_Basics/FindElementsTests.cs (hunk 1 above)
Comment (katherinaGo) [reply Selenium_Basics/FindElementsTests.cs:18]: Thanks. Replaced

File: Selenium_Basics/FindElementsTests.cs (hunk 2 above)
Comment (katherinaGo) [reply Selenium_Basics/FindElementsTests.cs:26]: changed. Thanks.

**Pull request merged**